addons/hr/migrations/11.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,17 @@
+---Fields in module 'hr'---
+hr           / hr.employee              / _inherits (False)             : DEL 
+hr           / hr.employee              / permit_no (char)              : NEW 
+hr           / hr.employee              / visa_expire (date)            : NEW 
+hr           / hr.employee              / visa_no (char)                : NEW
+# Nothing to do
+
+---XML records in module 'hr'---
+NEW ir.actions.act_window: hr.hr_config_settings_action
+NEW ir.ui.menu: hr.hr_menu_configuration
+NEW ir.ui.view: hr.hr_job_view_kanban
+NEW ir.ui.view: hr.res_config_settings_view_form
+DEL ir.values: hr.hr_employee_action_subordinate_hierachy_mutli
+DEL mail.message: hr.module_install_notification (noupdate)
+DEL res.groups: base.group_user (noupdate)
```

Comment:
noupdate records. You should create the `noupdate_changes.xml` file the same way as here: https://github.com/OCA/OpenUpgrade/pull/1175

Reply:
when i run 
python compare_noupdate_xml_records.py ../../OpenUpgrade/addons/hr ../addons/hr

i got this:

<?xml version='1.0' encoding='utf-8'?>
<odoo/>

Reply:
True. What you should do is remove the mail message in the post script. Because it is noupdate, it won't be removed automatically. Some records are better left in the database, but this message was never good for anything.

The res.groups is a false positive, of course. In this case it is the override that was removed from the module, not the group itself.

Reply:
Then again, as Pedro mentions below, the notification was already fired if this module is being upgraded.... So I agree with him, just make this a 'Nothing to do'

Reply:
Thank you @StefanRijnhart, i did